lighthouse-core/gather/drivers/driver.js

```diff
@@ -420,11 +421,21 @@ class Driver {
     return this.sendCommand('Network.setCacheDisabled', {cacheDisabled: true});
   }
 
-  forceUpdateServiceWorkers() {
-    // COMPAT: This command will trigger this registrationId error in Chrome 50 (51 undetermined):
-    //   "{"code":-32602,"message":"Missing or invalid 'registrationId' parameter"}"
-    return this.sendCommand('ServiceWorker.setForceUpdateOnPageLoad', {
-      forceUpdateOnPageLoad: true
+  clearDataForOrigin(url) {
+    const parsedUrl = parseURL(url);
+    const origin = `${parsedUrl.protocol}//${parsedUrl.hostname}`;
```

Comment:
need the port number too, if there is one. Something like [this](https://github.com/GoogleChrome/lighthouse/blob/a379d8f4f1633962163e55d5144ffe80bdbd4982/lighthouse-core/audits/service-worker.js#L30)